index.html

```diff
@@ -144,45 +144,112 @@ <h1 class="heading2">Project name goes here</h1>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
         </p>
         <button class="buttons-primary-dark" type="button">
-          Start collaboration &#160; <i class="fa fa-download" aria-hidden="true"></i>
+          Start collaboration &#160;
+          <i class="fa fa-download" aria-hidden="true"></i>
         </button>
         <ul class="cards-container">
-        <li class="blue about heading3"> Languages
-          <ul class="para2 side-li">
-            <li>JavaScript (ES5/ES6)</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-          <img class="icon-bg" src="images/ic_languages_mob.svg" alt="#">
-        </li>
-        <li class="red about heading3"> Frameworks
-          <ul class="para2 side-li">
-            <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>RSpec</li>
-            <li>Capibara</li>
-            <li>Selenium</li>
-          </ul>
-          <img class="icon-bg" src="images/ic_framework.svg" alt="#">
-        </li>
-        <li class="yellow about heading3"> Skills
-          <ul class="para2 side-li">
-            <li>Codekit</li>
-            <li>Codepen</li>
-            <li>Github</li>
-            <li>Gitlab</li>
-            <li>TErminal</li>
+          <li class="blue about heading3">
+            Languages
+            <ul class="para2 side-li">
+              <li>JavaScript (ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+            <img class="icon-bg" src="images/ic_languages_mob.svg" alt="#" />
+          </li>
+          <li class="red about heading3">
+            Frameworks
+            <ul class="para2 side-li">
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
+            <img class="icon-bg" src="images/ic_framework.svg" alt="#" />
+          </li>
+          <li class="yellow about heading3">
+            Skills
+            <ul class="para2 side-li">
+              <li>Codekit</li>
+              <li>Codepen</li>
+              <li>Github</li>
+              <li>Gitlab</li>
+              <li>TErminal</li>
+            </ul>
+            <img class="icon-bg" src="images/ic_skills.svg" alt="#" />
+          </li>
+        </ul>
+      </section>
+      <section class="form-container">
+        <img
+          class="nav logo icon"
+          src="images/image_geometry_1.svg"
+          alt="logo"
+        />
```

Comment:
- [ ] Let's make sure this image is a background image since it's only there for design reasons it's recommended to be as a background image